api/healthcheck/healthcheck_controller.go

```diff
@@ -19,45 +19,30 @@ package healthcheck
 import (
 	"net/http"
 
+	"github.com/Peripli/service-manager/pkg/health"
 	"github.com/Peripli/service-manager/pkg/log"
 	"github.com/Peripli/service-manager/pkg/util"
 	"github.com/Peripli/service-manager/pkg/web"
-	"github.com/Peripli/service-manager/storage"
 )
 
-
 // Controller platform controller
 type Controller struct {
-	Storage storage.Storage
+	Indicator health.Indicator
 }
 
 var _ web.Controller = &Controller{}
 
-var statusRunningResponse = map[string]interface{}{
-	"status": "UP",
-	"storage": map[string]interface{}{
-		"status": "UP",
-	},
-}
-
-var statusStorageFailureResponse = map[string]interface{}{
-	"status": "OUT_OF_SERVICE",
-	"storage": map[string]interface{}{
-		"status": "DOWN",
-	},
-}
-
 // healthCheck handler for GET /v1/monitor/health
 func (c *Controller) healthCheck(r *web.Request) (*web.Response, error) {
 	ctx := r.Context()
 	logger := log.C(ctx)
-	logger.Debug("Performing health check...")
-
-	if err := c.Storage.Ping(); err != nil {
-		logger.Debugf("storage.Ping failed: %s", err)
-		return util.NewJSONResponse(http.StatusServiceUnavailable, statusStorageFailureResponse)
+	logger.Debug("Performing healthResult check...")
```

Comment:
the indicators are named - should we log the names of the indicators used in the health check?